example/01-echo/libp2p_echo_client.cpp

```diff
@@ -35,8 +35,9 @@ int main(int argc, char *argv[]) {
   // create io_context - in fact, thing, which allows us to execute async
   // operations
   auto context = injector.create<std::shared_ptr<boost::asio::io_context>>();
-  context->post([host{std::move(host)}, &echo, argv] {
-    auto server_ma_res = libp2p::multi::Multiaddress::create(argv[1]);
+  context->post([host{std::move(host)}, &echo, argv] {  // NOLINT
```

Comment:
How do we decide where to put NOLINT? 

Can we document those kinds of exceptions in contributing.md ?

Reply:
When you are brave enough that clang-tidy suggestion can be ignored you can put it. Although I agree that we should explain it better